velox/connectors/hive/storage_adapters/s3fs/S3Util.h

```diff
@@ -163,7 +163,7 @@ inline std::string getRequestID(
           errorMsgPrefix,                                                                                                      \
           getErrorStringFromS3Error(error),                                                                                    \
           s3URI(bucket, key),                                                                                                  \
-          error.GetErrorType(),                                                                                                \
```

Comment:
@assignUser this change seems to be causing the linux-adapter failures.

Reply:
Thanks for investigating, do you think it makes sense to update the expected error message in the test or change something here? I tried to use what seemed to be the canonical way to do this but I am of course open to changes.

Reply:
From the [SDK](https://github.com/aws/aws-sdk-cpp/blob/3a536864870e9d4edb6d753ed4882e1ce229d1c8/generated/src/aws-cpp-sdk-s3/include/aws/s3/S3Errors.h#L16), it looks like `error.GetErrorType()` returns an enum.
We should be good with `static_cast<int>(error.GetErrorType())` here.